src/app/static/src/app/index.ts

```diff
@@ -30,7 +30,7 @@ export const app = new Vue({
     methods: {
         ...mapActions({loadBaseline: 'baseline/getBaselineData'}),
         ...mapActions({loadSurveyAndProgram: 'surveyAndProgram/getSurveyAndProgramData'}),
-        ...mapActions({loadModelRun: 'modelRun/getResult'}),
+        ...mapActions({loadModelRun: 'modelCalibrate/getResult'}),
```

Comment:
So I think you alluded to this in your comment, but what is supposed to happen when the user fits the model but does not calibrate and then refreshes? When I tested that scenario myself, I got a load error and the model fit was wiped.

Reply:
Oh good catch, wasn't expecting an error there..

Reply:
Yeah, the above doesn't work because it does need the modelRun result to be available for validate logic where it expects the modelRun step to be complete. So I've ended up making modelCalibrate state a readyState after all, with resultant boilerplate, and calling both these actions on page load..